### PR TITLE
Feature/uwp push build

### PR DESCRIPTION
**Pull request by achocron**
Can't merge until we update the nuget description fields

### CODE REVIEW

File: build.cake [hunk 1]
```diff
@@ -78,7 +78,9 @@ var MOBILECENTER_MODULES = new [] {
 	new MobileCenterModule("mobile-center-release.aar", "MobileCenter.framework.zip", "SDK/MobileCenter/Microsoft.Azure.Mobile", "MobileCenter.nuspec"),
 	new MobileCenterModule("mobile-center-analytics-release.aar", "MobileCenterAnalytics.framework.zip", "SDK/MobileCenterAnalytics/Microsoft.Azure.Mobile.Analytics", "MobileCenterAnalytics.nuspec"),
 	new MobileCenterModule("mobile-center-crashes-release.aar", "MobileCenterCrashes.framework.zip", "SDK/MobileCenterCrashes/Microsoft.Azure.Mobile.Crashes", "MobileCenterCrashes.nuspec"),
-	new MobileCenterModule("mobile-center-distribute-release.aar", "MobileCenterDistribute.framework.zip", "SDK/MobileCenterDistribute/Microsoft.Azure.Mobile.Distribute", "MobileCenterDistribute.nuspec")	
+	new MobileCenterModule("mobile-center-distribute-release.aar", "MobileCenterDistribute.framework.zip", "SDK/MobileCenterDistribute/Microsoft.Azure.Mobile.Distribute", "MobileCenterDistribute.nuspec"),
+	//																					when we have a pcl part, this should be changed vvvv (remove uwp extension)
```
Comment (guperrot) [inline build.cake]: Whats `vvvv` ?

Comment (guperrot) [inline build.cake]: Maybe use TODO in such comments

File: nuget/Microsoft.Azure.Mobile.Analytics.targets [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <mc-analytics-Platform Condition="'$(Platform)' == 'Win32'">x86</mc-analytics-Platform>
-    <mc-core-Platform Condition="'$(Platform)' == 'ARM'">arm</mc-core-Platform>
+    <mc-analytics-Platform Condition="'$(Platform)' == 'ARM'">arm</mc-analytics-Platform>
```
Comment (guperrot) [inline nuget/Microsoft.Azure.Mobile.Analytics.targets]: How did that work before on ARM?

File: nuget/MobileCenterPush.nuspec [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Microsoft.Azure.Mobile.Push</id>
+    <title>Mobile Center Push</title>
+    <authors>Microsoft Corporation</authors>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <summary>EDIT ME</summary>
+    <description>EDIT ME</description>
```
Comment (guperrot) [inline nuget/MobileCenterPush.nuspec]: Edit those.

File: nuget/WindowsMobileCenterAnalytics.nuspec [hunk 4]
```diff
@@ -17,17 +17,17 @@
   </metadata>
   <files>
 
-    <file src="nuget/Microsoft.Azure.Mobile.Analytics.targets"  target="build/uap10.0/Microsoft.Azure.Mobile.Analytics.targets"/>
+    <file src="nuget/Microsoft.Azure.Mobile.Analytics.targets"  target="build/uap10.0"/>
```
Comment (guperrot) [inline nuget/WindowsMobileCenterAnalytics.nuspec]: What is this change?

File: nuget/WindowsMobileCenterPush.nuspec [hunk 5]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Microsoft.Azure.Mobile.Push</id>
+    <title>Mobile Center Push</title>
+    <authors>Microsoft Corporation</authors>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <summary>EDIT ME</summary>
+    <description>EDIT ME</description>
```
Comment (guperrot) [inline nuget/WindowsMobileCenterPush.nuspec]: Edit texts

File: build.cake (hunk 1 above)
Comment (achocron) [inline build.cake]: Sorry those were supposed to be down arrows

File: nuget/Microsoft.Azure.Mobile.Analytics.targets (hunk 2 above)
Comment (achocron) [inline nuget/Microsoft.Azure.Mobile.Analytics.targets]: I don't really have a good answer

File: nuget/WindowsMobileCenterAnalytics.nuspec (hunk 4 above)
Comment (achocron) [inline nuget/WindowsMobileCenterAnalytics.nuspec]: I just removed the target file name to be consistent with the other nuspecs we have (if you only give the target directory it uses the source file name)

**Pull request merged**